### ISSUE TITLE
AVT 3 Number Input - VO does not announce value or increment when the up and down arrows used to input the number. 

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2179]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
With VoiceOver enabled Tab to number input. (Focus is on the up arrow)
Press the up arrow. Voice over does not announce the value or the increment number. 
Tab to the down arrow is the same.
Tab to the Number input and the value is announced.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.
